### ISSUE TITLE
Need a way to check sample completeness without downloading anything

### ISSUE THREAD

**Issue by dkirkby** [dkirkby/bossdata#10]
Perhaps a `--local-inventory` option to `bossfetch`?  The idea is to be able to issue a command like this:

```
% bossfetch --local-inventory sky.dat
1234 of 233807 observations are locally mirrored.
```

**Comment by dkirkby** [comment]
There is already a `--dry-run` option that stops after building the list of remote files, so it would be enough to add a loop over these files that reports the number of files that already have corresponding local mirrors before exiting.

**Comment by dkirkby** [comment]
Fixed in #37
